docs/src/main/asciidoc/dev-ui.adoc

```diff
@@ -15,37 +15,45 @@ include::_attributes.adoc[]
 This guide covers the Dev UI v2, which is the default from Quarkus 3 onwards.
 ====
 
-This guide covers the Quarkus Dev UI for xref:building-my-first-extension.adoc[extension authors].
```

Comment:
This sentence is very important and should stay at the top.
This guide is for extension developers.

I think what actually needs an adjustment is the title of the guide.

For the rest, I think I prefer the current content to your proposal as this guide has a very precise purpose and is not a general Dev UI guide.

Reply:
Thank you for your dedication and the effort you've put into your contributions, @allemas. After careful consideration, it appears that the reviewer has decided to maintain the existing content as it closely aligns with our current goals and guidelines. Therefore, it looks like we'll be closing this PR. However, your input is highly valued, and I plan to initiate a follow-up PR to update the title, incorporating aspects of your suggestions where applicable. We genuinely appreciate your understanding and look forward to your continued contributions to our project.